src/components/graph-view.js

```diff
@@ -1071,7 +1054,17 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
         }
 
         if (onUpdateNode) {
-          onUpdateNode(nodeMap.node);
+          let updatedNodes = null;
+
+          if (selectedNodes?.size > 0) {
+            updatedNodes = new Map(
```

Comment:
Must be missing something - why doesn't it work to set `updatedNodes = selectedNodes`? Also, why does this API call need to change with these bug fixes?

Reply:
Thanks. I think I had initially mistaken the selectedNodes for a nodeMapNode, which has a different structure.